justsendmail/mail.py

```diff
@@ -10,17 +10,19 @@
 import argparse
 import logging
 import os
-from typing import Optional, Dict
+from typing import Optional, Iterable, Dict, Union, List
```

Comment:
I'd prefer to see `import typing` here.

Reply:
`import typing as t`

704e60a087ee476d3c04288bfc0b800766c85a6e